src/blog/2024/05/flowfuse-on-robustel-eg5120.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Run FlowFuse on robustel EG5120
+subtitle: How to install FlowFuse Device Agent on robustel EG5120
+description: In this guide, we will discuss how to install FlowFuse Device agent on robustel EG5120.
+date: 2024-05-06
+authors: ["grey-dziuba"]
+image: /blog/2024/05/images/flowfuse-on-rubustel-eg5120-node-red.png
+tags:
+    - posts
+    - flowfuse
+    - node.js
+    - robustel
+    - eg5120
+---
+
+The [robustel EG5120](https://www.robustel.com/product/eg5120-industrial-edge-computing-gateway/) is a versatile gateway that facilitates robust connectivity for industrial IoT applications. Integrating this powerful hardware with FlowFuse not only enhances its capabilities but also simplifies the management and deployment process. In this guide, we’ll walk through the steps to integrate the robustel EG5120 with FlowFuse, making your IoT solutions more efficient and easier to manage.
+
+<!--more-->
+
+
+This blog is designed for IoT enthusiasts and professionals who are looking to enhance their industrial IoT applications. The [robustel EG5120](https://www.robustel.com/product/eg5120-industrial-edge-computing-gateway/), equipped with Linux-based Debian 11 supporting a wide variety of programming languages including Node.js, offers robust connectivity options. When combined with FlowFuse, this gateway becomes even more powerful, enabling seamless device management and deployment. 
+
+
+## robustel
+
+The robustel EG5120 supports multiple connectivity options including Ethernet, Wi-Fi, and cellular networks, which are essential for flexible deployments in various industrial scenarios. Its built-in support for Bluetooth, cellular connectivity, RS232, RS485, and Modbus facilitates seamless integration with a wide array of IoT devices and services. This blog will guide you through using FlowFuse to effectively manage your Node-RED instance, enhancing both the security and scalability of your IoT applications.
+
+## Migration
+
+If you're looking to migrate your Node-RED instance, follow the steps in [this migration guide](/docs/migration/node-red-tools/) to back up and transfer your configurations safely. By incorporating the FlowFuse device agent, we aim to replace the  Node-RED runtime, thus providing enhanced monitoring, automated backups, and improved management capabilities. This transition will make your deployments more secure and resilient.
```

Comment:
As it's not installed by default, why migrate?

Reply:
Go point.  This was before I learned that I was working from a preconfigured device.  Some may still be using node-red as it does come preinstalled on NCD.io's devices.

Reply:
But they don't google Robustel EG5120 Node-RED? Not sure why this comment was resolved but went unhandled. 

I'd be fine with an article outlining how NCD works that's more or less the same as this one, but this paragraph feels misplaced.